weather_mv/loader_pipeline/util.py

```diff
@@ -85,14 +85,19 @@ def to_json_serializable_type(value: t.Any) -> t.Any:
     # Note: The order of processing is significant.
     logger.debug('Serializing to JSON')
 
-    if pd.isna(value) or value is None:
+    # pd.isna() returns ndarray if input is not scalar therefore checking if not ndarray.
```

Comment:
Comment is a bit stale.

Reply:
updated in https://github.com/google/weather-tools/pull/327/commits/af655e7d4c63b3dc2e39e87271f5528016981d50